aws/table_aws_spot_price_history.go

```diff
@@ -0,0 +1,151 @@
+package aws
+
+import (
+	"context"
+	"errors"
+	"github.com/aws/aws-sdk-go-v2/service/ec2"
+	"github.com/aws/aws-sdk-go-v2/service/ec2/types"
+	"github.com/turbot/steampipe-plugin-sdk/v4/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin/transform"
+	"time"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsSpotPriceHistory(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_spot_price_history",
```

Comment:
@aminvielledebatAtBedrock Do you know if this provides the same info as https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/ec2_spot_price? If so, I'm inclined to change this table name to match Terraform's, so `aws_ec2_spot_price`. In general, we start by asking What Would Terraform Do, and then go from there.